### ISSUE TITLE
Copyright License issue on formatter.py

### ISSUE THREAD

**Issue by billyswong** [shlomif/PySolFC#331]
It seems Debian and Ubuntu are being reluctant to package the up-to-date PySolFC in their .deb repository.
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1008708 (See message 21)
https://bugs.launchpad.net/ubuntu/+source/pysolfc/+bug/1967793 (See comment 8)

Is the concern of Athos Ribeiro a real concern that this project need to fix?

**Comment by joeraz** [comment]
So here's my perspective on this (not a lawyer, just a developer).

First, the formatter.py file was not re-licensed as CC0 - such a re-license would not be possible, as CC0 is a less restrictive license than the Python license.

The only changes made to the formatter.py file (aside from the additional copyright notice at the top) were just adding and removing whitespace, and the removal of the deprecation warning.  I ran a code compare tool to make sure of this.  I believe @shlomif intended to make these changes CC0, but the file as a whole would still be under the Python license (CC0 is public domain, so CC0 code can be used anywhere without restriction).  And the license for Python is GPL-compatible, so it can be used in PySolFC, which is GPLv3.

No copyright notices were removed from the file, but we'll see if we can update the copyright notice for clarity, and to ensure everything's legitimate.
